gef-ui/server/routes/mandatory-criteria.js

```diff
@@ -0,0 +1,13 @@
+import express from 'express'
+import { 
+  getMandatoryCriteria,
+  validateMandatoryCriteria
+} from '../controllers/mandatory-criteria'
+import validateToken from './middleware/validate-token'
+
+const router = express.Router()
+
+router.get('/mandatory-criteria', validateToken, (req, res) => getMandatoryCriteria(req, res))
+router.post('/mandatory-criteria', validateToken, (req, res) => validateMandatoryCriteria(req, res))
```

Comment:
just need to add some tests for the routes e.g:
- https://github.com/foundry4/dtfs2/blob/master/trade-finance-manager-ui/server/routes/index.test.js
- https://github.com/foundry4/dtfs2/blob/master/trade-finance-manager-ui/server/routes/case/index.test.js

Reply:
Done. Couldn't mock the anonymous function and its controller, so ended up using 'expect.any(Function)'. If you have a solution please let me know

Reply:
I think if it's changed to 

```
router.get('/mandatory-criteria', validateToken, getMandatoryCriteria);
```

Then the test can assert the passed controller:

```
expect(get).toHaveBeenCalledWith('/mandatory-criteria', validateToken, getMandatoryCriteria);
```

?

Reply:
doesn't seem to work as i need to pass both the req and res parameters to the controllers

Reply:
OK cool would be nice to do, not a blocker for me.

The req/res should get passed to the controller automagically, I suspect the mock `validateToken` might not be passing this in a `next()` call, in the test

Can improve later...